README.md

```diff
@@ -28,6 +28,10 @@ If you like the project, you can become a sponsor at [Open Collective](https://o
 
 <a href="https://opencollective.com/excalidraw#category-CONTRIBUTE" target="_blank"><img src="https://opencollective.com/excalidraw/tiers/backers.svg?avatarHeight=32"/></a>
 
+Last but not least, these companies are sponsoring Excalidraw by offering their services:
+
+[![Vercel](./images/vercel.svg)](https://vercel.com) [![Sentry](./images/sentry.svg)](https://sentry.io)
```

Comment:
```suggestion
[![Vercel](images/vercel.svg)](https://vercel.com) [![Sentry](images/sentry.svg)](https://sentry.io)
```

Reply:
maybe `assets` to be more generic for future stuff?

Reply:
I think we can add images under `.github` since it's part of the GitHub readme front page :)